spec/features/apo_view_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+feature 'APO views' do
+
+  context 'with viewing user' do
+    let(:druid) { 'druid:hv992ry2431' }
+    let(:item) { instantiate_fixture(druid, Dor::AdminPolicyObject) }
+    before :each do
+      # use a viewing user that depends on workgroup roles for access
+      view_user
+      allow(@view_user).to receive(:is_viewer).and_return(false)
+      allow(Dor).to receive(:find).with(druid).and_return(item)
+    end
+
+    it 'allows a user of an authorized workgroup to view an APO' do
+      # Ensure all user authority methods return false and the
+      # user belongs to at least one authorized workgroup.
+      dor_viewer_roles = ["dor-viewer", "sdr-viewer"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.